backend/libraries/ballerina-core/business rule engine/parser/expr.fs

```diff
@@ -251,3 +254,108 @@ module Expr =
           )
       }
       |> state.MapError(Errors.HighestPriority)
+
+    static member UnparseValue(value: Value) : Sum<JsonValue, Errors> =
+      sum {
+        match value with
+        | Value.ConstBool b -> JsonValue.Boolean b
+        | Value.ConstInt i -> JsonValue.Number(decimal i)
+        | Value.ConstString s -> JsonValue.String s
+        | Value.ConstGuid _ -> return! sum.Throw(Errors.Singleton "Error: ConstGuid not implemented")
+        | Value.Unit -> return! sum.Throw(Errors.Singleton "Error: Unit not implemented")
+        | Value.Lambda(variable, expression) ->
```

Comment:
to be precise: this is a parameter (see also the naming in the json record you construct)

Reply:
Using variable as:
- TAPL uses it
- the type is `| Lambda of VarName * Expr`

Reply:
Done